addons/account/wizard/account_move_send.py

```diff
@@ -332,6 +325,28 @@ def _format_error_html(self, error):
     # BUSINESS ACTIONS
     # -------------------------------------------------------------------------
 
+    @api.model
+    def action_open_partners_without_email(self, res_ids=None):
+        partners = self.env[self._name].browse(res_ids).move_ids.mapped("partner_id")
```

Comment:
> **Claire**:
> 1.  Why not just make the action not @api.model and use self.move_ids here ?
> 2. This browse would therefore not be necessary.
    This is retrieving all partners from all moves, we should only display partners without email :
    self.move_ids.filtered(lambda move: not move.partner_id.email)

Reply:
1. In stable, yes, we should / could. But I knew this wasn't gonna be merged before the freeze.
2. Done.